frontend/src/utils/RequestHelper.ts

```diff
@@ -28,15 +29,25 @@ export default class RequestHelper {
     return { headers: { Authorization: `Bearer ${userStore().accessToken}` } }
   }
 
-  static async get(url: string): Promise<AxiosResponse> {
-    return await axios.get(url, this.getHeaders())
+  static async get(url: string): Promise<AxiosResponse | void> {
+    return await axios.get(url, this.getHeaders()).catch((error): void => {
+      if (error.response.status === HttpStatusCode.Forbidden) {
+        userStore().clearFields()
+        router.push('/login')
+      }
```

Comment:
We should not mix async/await and then/catch (CPS).
Instead assign the request response to a variable and check it later.
`const res = await axios.get(....)
if (res.status) ecc`.
Last point, is not very DRY, you can make a reusable private method to make the control